### PR TITLE
sys-process/acct: initd implement checkpath, #603606

### PR DESCRIPTION
**Pull request by wimmuskee**
Gentoo-Bug: https://bugs.gentoo.org/603606

### CODE REVIEW

File: sys-process/acct/files/acct.initd [hunk 1]
```diff
@@ -11,10 +11,7 @@ depend() {
 
 checkconfig() {
 	export ACCT_LOG=${ACCT_LOG:-/var/account/pacct}
-	if [ ! -e "${ACCT_LOG}" ] ; then
-		touch "${ACCT_LOG}"
-		chmod 600 "${ACCT_LOG}"
-	fi
+	checkpath -f -m 600 "${ACCT_LOG}"
```
Comment (SoapGentoo) [inline sys-process/acct/files/acct.initd]: You're updating the init file, but how can you ensure that the updated file arrives on users' machines? You'll need to revbump the latest stable and unstable ebuilds.

Comment (wimmuskee) [inline sys-process/acct/files/acct.initd]: and leave the existing ebuilds as well I presume?

Comment (SoapGentoo) [inline sys-process/acct/files/acct.initd]: yes, just latest of stable and unstable, use `eshowkw` for an easy overview (i.e. `6.5.5-r3` and `6.6.2`)

Comment (wimmuskee) [inline sys-process/acct/files/acct.initd]: done, and rebased the changes to one commit

**Comment by robbat2** [comment]
merged

**Pull request closed without merging**